circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 0.10.40
+dependencies:
+  cache_directories:
+    - node_modules
+test:
+  override:
+    - npm run ci
```

Comment:
As you are running istanbul for coverage I'd suggest adding a `post` step here and copying the coverage reports to the `CIRCLE_ARTIFACTS` folder